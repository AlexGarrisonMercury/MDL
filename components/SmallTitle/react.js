import React from 'react'
import PropTypes from 'prop-types'
import classNames from 'classnames/bind'
import style from './style.module.scss'

const cx = classNames.bind(style)

export const SmallTitle = ({ children }) => <h2 className={cx('small-title')}>{children}</h2>

export default SmallTitle

SmallTitle.propTypes = {
  children: PropTypes.string.isRequired,
}
