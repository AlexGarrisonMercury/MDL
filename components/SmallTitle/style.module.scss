@import "styles/toolbox";

.small-title {
  font-size: 14px;
  font-weight: 300;
  text-transform: uppercase;
  margin: $space-s 0;
  color: $color-night;

  &:first-child {
    margin-top: 0;
  }
}
