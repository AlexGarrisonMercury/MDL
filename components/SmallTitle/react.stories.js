import React from 'react'
import { storiesOf } from '@storybook/react'
import SmallTitle from './react'
import Readme from './react.README.md'

storiesOf('SmallTitle', module)
  .addParameters({ readme: { sidebar: Readme } })
  .add('default', () => <SmallTitle>Small Title</SmallTitle>)
