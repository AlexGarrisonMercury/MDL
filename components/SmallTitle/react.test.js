import React from 'react'
import { shallow } from 'enzyme'
import SmallTitle from './react'

describe('SmallTitle', () => {
  it('should render without crashing', () => {
    expect(shallow(<SmallTitle>Title</SmallTitle>)).toMatchSnapshot()
  })
})
