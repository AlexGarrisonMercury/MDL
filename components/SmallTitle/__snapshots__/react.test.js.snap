// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SmallTitle should render without crashing 1`] = `
<h2
  className="small-title"
>
  Title
</h2>
`;
